packages/sampler/stories/qa-stories/Scroller.js

```diff
@@ -24,6 +25,29 @@ const
 		horizontalScrollbar: ['auto', 'hidden', 'visible']
 	};
 
+class ScrollerWithResizable extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			more: false
+		};
+	}
+
+	handleClick = () => {
+		this.setState(prevState => ({more: !prevState.more}));
+	}
+	render () {
+		return (
+			<Scroller direction="vertical" verticalScrollbar="visible">
```

Comment:
We don't need `direction="vertical"` here.

Reply:
done.